### DISCUSSION TITLE
remainingTime Options

### DISCUSSION THREAD

**Discussion by ShahriarKh** [persian-tools/persian-tools#149] [Ideas]
It will be really great if the library supports different ways of showing remaining time out-of-the-box, based on the passed arguements.
### Currently:
```js
remainingTime("2023-05-14T13:35:59Z").toString() // ۱ سال و ۱ ماه و ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه 
```
### New:
```js
remainingTime("2023-05-14T13:35:59Z").toString({ years, hours }) // ۱ سال و ۳ ساعت  
```

I know it's possible to do it manualy like this:
```jsx
const { years, hours } = remainingTime("2023-05-14T13:35:59Z");
return <span>{years} سال و {hours} ساعت</span>
```
But this can also be an option:
```jsx
return <span>{remainingTime("2023-05-14T13:35:59Z").toString({ years, hours })}</span>
```
This is more readable in many places, especially in text editors that do not handle mixed RTL, LTR correctly.

**Comment by ali-master** [comment]
Hi there,

I agree with you and I appreciate it if you help the Persian tools by implementing the idea by yourself 😊

**Reply by ShahriarKh** [reply]
I will take a look
